### ISSUE TITLE
Lets my 1st model train on A ,B,C class...In next time i want to train My pre-train 1st model by adding D class ...(Note: In second time training I do not want to train and mix all data...i just want to train new D class). Final model must detect A, B, C, D class.

### ISSUE THREAD

**Issue by TapendraBaduwal** [Deci-AI/super-gradients#922]
issue faced in contineous learning....Lets my 1st model train on A ,B,C class...In next time i want to train My pre-train 1st model by adding D class ...(Note: In second time training I do not want to train and mix all data...i just want to train new D class). Final model must detect A, B, C, D class.

In next time without mixing the A ,B,C and D class data...I want to Load pre-train model and just load new unseen data D class and train.
My final new model must detect A,B,C,D class together.

Give me best Solution for that with supportive code flow.
